init-tools.sh

```diff
@@ -11,10 +11,8 @@ export __BUILDTOOLS_USE_CSPROJ=true
 __BUILD_TOOLS_PACKAGE_VERSION=$(cat $__scriptpath/BuildToolsVersion.txt)
 __DOTNET_TOOLS_VERSION=$(cat $__scriptpath/DotnetCLIVersion.txt)
 __BUILD_TOOLS_PATH=$__PACKAGES_DIR/microsoft.dotnet.buildtools/$__BUILD_TOOLS_PACKAGE_VERSION/lib
-__PROJECT_JSON_PATH=$__TOOLRUNTIME_DIR/$__BUILD_TOOLS_PACKAGE_VERSION
-__PROJECT_JSON_FILE=$__PROJECT_JSON_PATH/project.json
-__PROJECT_JSON_CONTENTS="<Project Sdk=\"Microsoft.NET.Sdk\"><PropertyGroup><TargetFramework>netcoreapp1.0</TargetFramework><DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences></PropertyGroup><ItemGroup><PackageReference Include=\"Microsoft.DotNet.BuildTools\" Version=\"$__BUILD_TOOLS_PACKAGE_VERSION\" /></ItemGroup></Project>"
-__INIT_TOOLS_DONE_MARKER=$__PROJECT_JSON_PATH/done
+__INIT_TOOLS_RESTORE_PROJECT=$__scriptpath/init-tools.msbuild
```

Comment:
We can get rid of some of it, but I think there will still be quite a bit left over. Most of the script is locating and downloading the CLI archive, or running crossgen, etc. All we really do with init-tools.msbuild is restore it and then call the "inner init-tools" script inside the packages directory.